app/main.py

```diff
@@ -1,13 +1,30 @@
 def format_linter_error(error: dict) -> dict:
-    # white your code here
-    pass
+    return {"line": error["line_number"],
+            "column": error["column_number"],
+            "message": error["text"],
+            "name": error["code"],
+            "source": "flake8"}
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # white your code here
-    pass
+    return {"errors": [
+        {"line": errors[i]["line_number"],
+         "column": errors[i]["column_number"],
+         "message": errors[i]["text"],
+         "name": errors[i]["code"],
+         "source": "flake8"}
```

Comment:
You have implemented "format_linter_error" above, why did you duplicate code from that function?

Reply:
okay. but I thought the point was to realize them from scratch

Reply:
No, main concept of function is to encapsulate code inside and reuse it

Reply:
thx, sorry, it's my mistake